### ISSUE TITLE
Syncribullet

### ISSUE THREAD

**Issue by aliyss** [danamag/stremio-addons-list#267]
### Addon Manifest URL

https://56bca7d190fc-syncribullet.baby-beamup.club/manifest.json

### Addon Description

* Info
Currently the addon live synchronizes with following services:
- Anilist
- Simkl

Additionally it adds the services as Catalogs to Stremio

** End Goal
To implement a two way sync over multiple services with Stremio acting as the _Sender_ and the watchlists acting as the _Receivers_.

### Language of Content

None

### Choose Labels

- [X] anime
- [ ] asian drama
- [ ] bollywood
- [X] cartoons
- [ ] debrid support
- [ ] http streams
- [ ] live tv
- [X] metadata
- [X] misc
- [X] movies
- [ ] music
- [ ] nsfw
- [ ] podcasts
- [ ] radios
- [ ] subtitles
- [ ] torrents
- [X] tv shows
- [ ] video games

**Comment by aliyss** [comment]
Just to clarify... It is heavily recommended to configure the addon before installing it. Otherwise it won't do much. ^^

**Comment by Jleep2017** [comment]
Unsubscribe

**Comment by ericvlog** [comment]
hello, thanks for wonderful addons.

It have some  bug on android tv stremio , the main page catalog won't load poster, but the background image is okay.

on stremio version android & windows it can run very well , just android tv stremio have problem

Thanks regard.

**Comment by aliyss** [comment]
> > Just to clarify... It is heavily recommended to configure the addon before installing it. Otherwise it won't do much. ^^
> 
> If the addon _requires_ configuration you can set [this](https://github.com/Stremio/stremio-addon-sdk/blob/master/docs/api/responses/manifest.md?plain=1#L103C1) in the `manifest.json` to force it in the Stremio client.

Yep. is done. Just in case someone is sharing it directly with another person

**Comment by Djlilyazii** [comment]
Will this work with the add on Anime Catalogs ?

[https://1fe84bc728af-stremio-anime-catalogs.baby-beamup.club/configure](url)

**Comment by aliyss** [comment]
> Will this work with the add on Anime Catalogs ?

Now it should

**Comment by Dude138** [comment]
That's works Great for Anilist

**Comment by Djlilyazii** [comment]
> > Will this work with the add on Anime Catalogs ?
> 
> Now it should

thanks

**Comment by Djlilyazii** [comment]
how do we donate to you ?

**Comment by aliyss** [comment]
> how do we donate to you ?

@Djlilyazii If you really want to, I've added the links at the bottom of the [README](https://github.com/aliyss/syncribullet?tab=readme-ov-file#support). Thanks for the initiative. Also ping me on the discord server... I'll give a role, with no additional perks than bragging rights.

**Comment by Djlilyazii** [comment]
> > how do we donate to you ?
> 
> [@Djlilyazii](https://github.com/Djlilyazii) If you really want to, I've added the links at the bottom of the [README](https://github.com/aliyss/syncribullet?tab=readme-ov-file#support). Thanks for the initiative. Also ping me on the discord server... I'll give a role, with no additional perks than bragging rights.

<img width="546" alt="Image" src="https://github.com/user-attachments/assets/a8ddf319-f670-4db4-b27a-cbf92e6e44b0" />
